main_routers/music_router.py

```
@@ -65,6 +23,152 @@ def _patch_pyncm_async() -> None:
+                    'Cache-Control': 'public, max-age=21600',
+                    'X-Cache': 'MISS',
+                    'Accept-Ranges': 'bytes'
+                }
+            )
+
+    except httpx.HTTPStatusError as e:
+        logger.error(f"[Music Proxy] HTTP错误: {e.response.status_code}")
+        return JSONResponse(content={"success": False, "error": f"请求失败: {e.response.status_code}"}, status_code=e.response.status_code)
+    except httpx.TimeoutException:
+        return JSONResponse(content={"success": False, "error": "请求超时"}, status_code=504)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)